### PR TITLE
updated docs, new def that displays an information message before starting the CLI, drives check on Windows

### PR DESCRIPTION
**Pull request by NataliaBondarenko**
Hello!

changelog:
- updated docs with new images
The plan is as follows: 1 example with default table, 1 example with sort-alpha, 1 example with list of file paths, 1 eample with preview and sizes, 1 example with total counting of files.
Please, add a couple of matching images from the Mac OS to the https://github.com/victordomingos/Count-files/tree/master/docs/source/_static folder.
You can use any path, not just ~/Documents. You can choose any of those that already exist, or add a new one. I do not have beautiful images :)
- added def show_start_message in viewing_modes.py
- a condition is added to not check the is_hidden_file_or_dir(path), if it is a local drive (on Windows) in main.py

In general, we can already change the source for the Read the Docs.

### CODE REVIEW

File: count_files/__main__.py [hunk 1]
```diff
@@ -171,31 +173,38 @@ def main_flow(*args: [argparse_namespace_object, Union[bytes, str]]):
...
     if not os.path.exists(location):
-        parser.exit(status=1, message=f'The path {location} does not exist, or there may be a typo in it.')
+        parser.exit(status=1, message=f'The path {location} '
+                                      f'does not exist, or there may be a typo in it.')
 
     if not include_hidden and is_hidden_file_or_dir(location):
-        parser.exit(status=1, message=f'\nNot counting any files, because {loc_text[2:]} has hidden folders.\n'
-                                      f'Use: python -m count_files {args.path} --all')
+        # skip check if path is a local drive
+        if platform.startswith('win') and len(Path(location).parents) == 0:
```
Comment (victordomingos) [inline count_files/__main__.py]: Why are we skipping this check on windows?

File: count_files/utils/file_handlers.py [hunk 2]
```diff
@@ -167,6 +167,9 @@ def is_hidden_file_or_dir return True, even if it has visible final file/folder.
     Windows: when used separately, the function returns False if the path does not exist.
     Linux, Mac OS: when used separately, the result depends on the presence of a "/." in the path.
 
+    Windows: if filepath is a local drive it returns True.
+    is_hidden_file_or_dir('C:/') True , ...('C:') False
+    is_hidden_file_or_dir('D:/') True, ...('D:') True
```
Comment (victordomingos) [inline count_files/utils/file_handlers.py]: I don't get it. Why should the hidden status be considered different for local or non-local drives?

File: count_files/__main__.py (hunk 1 above)
Comment (NataliaBondarenko) [reply count_files/__main__.py]: This function with `ctypes.windll.kernel32.GetFileAttributesW` retrieves file system attributes for a specified file or directory. It returns True for C:\, D:\, ...  But in fact, these drives are not hidden. To count the files you always need to use, the `--all` argument.

File: count_files/utils/file_handlers.py (hunk 2 above)
Comment (NataliaBondarenko) [reply count_files/utils/file_handlers.py]: I don't get it too.

**Pull request merged**